src/transformations/extent.jl

```diff
@@ -6,19 +6,24 @@ calculating and adding an `Extents.Extent` to all objects.
 
 This can improve performance when extents need to be checked multiple times.
 """
-embed_extent(x; kw...) = apply(AbstractTrait, x; kw...)
+embed_extent(x; kw...) = apply(extent_applicator, GI.AbstractTrait, x; kw...)
```

Comment:
We also might want to specify here what keywords are possible. Cross-referencing with the with `primitives.jl`, I can see that it they are: `crs`, `calc_extent`, and `threaded`... however, down below we recalculate crs and we are obviously calculating extent so we might what to specify why we use apply.

Reply:
Ahh probably this can all be written just using one line - `apply(identity, PointTrait, x; `calc_extent=true)` !!!

`calc_extent` already preopagates the extents efficiently.

And then the `threaded` and `crs` keywords make sense. We can just block the `calc_extent` keyword.  

lol